vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -140,7 +140,18 @@ var _ = RecommenderE2eDescribe("Checkpoints", func() {
 		_, err := vpaClientSet.AutoscalingV1().VerticalPodAutoscalerCheckpoints(ns).Create(context.TODO(), &checkpoint, metav1.CreateOptions{})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-		time.Sleep(15 * time.Minute)
+		fmt.Println("Sleeping for up to 15 minutes...")
```

Comment:
Can we use logger instead of println?

Reply:
I think we can. I'm not sure what is recommended for tests though. There seems to be a mix of Print and klog.
I guess klog makes sense?

Reply:
Fixed in 56afad215c